test/StandardSqlFormatterTest.js

```diff
@@ -1,402 +1,366 @@
 import sqlFormatter from "./../src/sqlFormatter";
```

Comment:
It looks like my linter re-formatted this file a bunch. The changes start at the bottom of the file